### ISSUE TITLE
0.11.0 crashes on Arch Linux

### ISSUE THREAD

**Issue by jacobeva** [SkaceKamen/sqflint#21]
![image](https://user-images.githubusercontent.com/52288074/103174193-fda90800-4857-11eb-907b-df5053ef60c9.png)
I am not experienced with java, however I believe this means you are not importing the commons jar correctly?
0.10.1 works fine.

**Comment by SkaceKamen** [comment]
That's weird, it works fine on my machine. Which version of Java are you using?

**Comment by jacobeva** [comment]
OpenJRE 14. I tried it on OpenJRE 8 instead and it still doesn't work, same error.

**Comment by jacobeva** [comment]
Do you use the commons library in 0.10.1?

**Comment by SkaceKamen** [comment]
Yep, it's something sqflint was using for some time now. I've used a newer
NetBeans to build the jar for this version, so maybe there's some issue
with that. I'll compare the builds and report back.

Dne po 28. pro 2020 22:13 uživatel x-00010 <notifications@github.com>
napsal:
